nexus/nexus-strawman.md

```diff
@@ -0,0 +1,358 @@
+**Contents:**
+
+- [Terminology](#terminology)
+- [User flows](#user-flows)
+  - [Hosting a Nexus Service](#hosting-a-nexus-service)
+  - [Consuming a Nexus Service from Temporal](#consuming-a-nexus-service-from-temporal)
+- [Concepts](#concepts)
+  - [Nexus Instantiation](#nexus-instantiation)
+  - [Nexus CRUD](#nexus-crud)
+  - [Nexus Request Handler hosting](#nexus-request-handler-hosting)
+  - [Calling Nexus from Temporal](#calling-nexus-from-temporal)
+  - [Nexus Worker](#nexus-worker)
+
+## Terminology
+
+**Nexus Service Instance:** Formally registered Nexus Service with a Nexus backend (eg: Temporal).  Nexus Instances must always have a unique identifier.
+
+**Nexus Request:** Unit of work which is run as a result of a request reaching the Nexus endpoint. Nexus Requests are developed and operated similar to WorkflowTasks and ActivityTasks (eg: run and hosted on Nexus Workers). Within a Nexus Request Handler, developers can transform incoming payloads, make routing decisions and more.
+
+**Nexus Request Handler:** Source code which is designed to handle incoming Nexus Requests. Nexus Request Handler is similar to a Workflow or Activity definition and is hosted by a Nexus Worker.
+
+**Nexus Worker:** A special purpose Worker that pulls Nexus Requests for its registered Nexus Instances from the Server and executes them using associated Nexus Request Handlers.
+
+## User flows
+
+### Hosting a Nexus Service
+
+1. User instantiates Nexus Service Instance using provided `nexus` CLI commands (also possible via SDK or API) and receives an Instance URI. Nexus is now able to receive requests to its frontend endpoint (though they don't get processed until step 4).
+1. User defines a Nexus Request Handler that will handle the processing of Nexus Requests for the instantiated Nexus Service Instance.
+1. User hosts their Nexus Request Handler using a Nexus Worker. They must specify the Nexus Service Instance URI when starting the Nexus Worker.
+
+    // TODO (Chad): show how registration would work for generic handler
```

Comment:
You have that below in the "Nexus Request Handler hosting"

Reply:
```suggestion
```